src/operator/softmax_output.cc

```diff
@@ -71,7 +71,27 @@ MXNET_REGISTER_OP_PROPERTY(SoftmaxOutput, SoftmaxOutputProp)
   The provided label can be a one-hot label array or a probability label array.
 
   - If the parameter `use_ignore` is ``true``, `ignore_label` can specify input instances
-    with a particular label to be ignored during backward propagation.
+    with a particular label to be ignored during backward propagation. **This is disabled when
+    softmax** `output` **has same shape with** `label`.
```

Comment:
* Is it right to say that this is "disabled"? Perhaps makes more sense to say "This has no effect when ...". 
* Can we remove the  asterisks (*)?
* Please change "has same shape with" to "has same shape as"